src/feature/RansacFeatureSetMatcher.cpp

```diff
@@ -123,5 +123,8 @@ double RansacFeatureSetMatcher::matchSets(const std::vector<InterestPoint *> &re
 	pointCorrespondences[i] = std::make_pair(correspondences[i].first->getPosition(), correspondences[i].second->getPosition());
     }
     compute2DPose(pointCorrespondences, transformation);
-    return verifyHypothesis(reference, data, transformation, correspondences);
+    double score = verifyHypothesis(reference, data, transformation, correspondences);
+    // Modify the score to be the sum of the errors of the inliers only
+    score -= (data.size()-correspondences.size())*m_acceptanceThreshold;
```

Comment:
I'd try to avoid changing the logic here. Maybe you can add an option to `verifyHypothesis`, so it doesn't scored the outliers with `m_acceptanceThreshold`, and make it default to the previous behavior.